modules/misc/news.nix

```diff
@@ -991,6 +991,13 @@ in
           A new module is available: 'programs.keychain'.
         '';
       }
+
+      {
+        time = "2019-02-20T15:37:15+00:00";
+        message = ''
+          A new service is available: 'services.mpDris2'.
```

Comment:
I think `mpdris2` is better since it is consistent with the package and file names.

Reply:
Fixed.